raiden/smart_contracts/Registry.sol

```diff
@@ -20,6 +20,10 @@ contract Registry {
         _;
     }
 
+    /// @notice addAsset to register a new ERC20 token
```

Comment:
Same as above, in all of the functions of this file. Do not add function name to the `@notice` tag and if `@dev` and `@notice` have no difference, then remove `@dev`